### ISSUE TITLE
[A11y] Changes in month are not announced to the user

### ISSUE THREAD

**Issue by DmitriiKasnitskii** [wix/react-native-calendars#1541]
## Description

Month changing should be announced for a user. Make sure that the name of the newly opened month is announced to the user, for example, "April 2021". Check that the focus remains on the selected date. 
Therefore, the reader should announce the newly opened month and the currently selected date.

## Expected Behavior

Once the date is selected the month of the calendar is changed to April 2021 and the change in the month is announced to the user. The focus remains on the "2 April" date. 

## Observed Behavior

Once the date is selected the month of the calendar is changed to April 2021 but the change is not announced to the user. The focus moves to the "30 April" date.

## Environment

Please run these commands in the project folder and fill in their results:

* `react-native-calendars`: ^1.505.0
* `react-native`: 0.63.2

Also specify:

1. iOS 14.4
